docs/level-18.md

```diff
@@ -10,7 +10,9 @@ import LoadClue from '@site/image-generator/yml/level-18/load-clue.yml';
 import PriorityFinesseSpecial from '@site/image-generator/yml/level-18/priority-finesse-special.yml';
 import TrustFinesse from '@site/image-generator/yml/level-18/trust-finesse.yml';
 
-- Level 18 strategies should only be learned if you have 320+ games of experience with the group.
+- This level introduces *Priority*, which requires players to pay attention to the order that cards are played in.
+- This can take a while to internalize, because cues from *Priority* are easy to "miss".
```

Comment:
cues or clues?

Reply:
I did mean "cues". Would a different word be better? ("signals"?)

Reply:
signals is better

Reply:
Sounds good